R/get-param.R

```diff
@@ -0,0 +1,128 @@
+#' Return theta values as a named vector
+#'
+#' @param .summary  A `bbi_nonmem_summary` or `bbi_summary_list` object.
+#' @export
+get_theta <- function(.summary){
+
+  if (inherits(.summary, "bbi_nonmem_summary")) {
+    .summary <- list(.summary)
+    check_model_object_list(.summary, .mod_types = c(NM_SUM_CLASS))
+  }else if(inherits(.summary, "bbi_summary_list")){
+    check_model_object(.summary, .mod_types = c("bbi_summary_list"))
```

Comment:
Given 1) `NM_SUM_CLASS` and 2) a `SL_CLASS <- "bbi_summary_list"` mapping exists, should `SL_CLASS` be used here?

Reply:
Oh I take the above comment back, will make the change to `SL_CLASS`